src/shared/components/Navigation/Navigation.scss

```diff
@@ -0,0 +1,61 @@
+.c-nav ul {
+	margin: 0;
+	padding: 0;
+}
+
+.c-nav li {
+	display: inline-block;
+	margin-right: 0.4rem;
+}
+
+.c-nav__item {
```

Comment:
Make it more sass-y.

```
.c-nav {
	&__item {
		&:hover, &:focus {

		}

		&--i {

		}
	}
}
```

Reply:
Dit zal wss overschreven worden door andry zijn aanpassingen